src/html/index.html

```diff
@@ -15,6 +15,11 @@
       <i class="material-icons">home</i> Home
     </a>
   </li>
+  <li ng-if="env.schemas.service">
+    <a href="#!/services" class="waves-effect">
+      <i class="material-icons">dvr</i> Services
+    </a>
+  </li>
   <li>
     <a href="#!/apis" class="waves-effect">
       <i class="material-icons">apps</i> APIs
```

Comment:
If `env.schemas.service` exists, I think you can add "(deprecated)" after "APIs".

Reply:
Done.